### PR TITLE
User story48

### PR DESCRIPTION
**Pull request by zachlp2016**
Added tests that the item can be created.  Tested all edge cases.

### CODE REVIEW

File: app/controllers/merchants/items_controller.rb [hunk 1]
```diff
@@ -4,9 +4,58 @@ def index
...
+
+  private
+
+  def items_params
+    params.require(:item).permit(:name, :price, :description, :image, :inventory )
+  end
+
+  def verify_price(params)
+    if params["item"]["price"].to_i < 0
+      return false
```
Comment (WHomer) [inline app/controllers/merchants/items_controller.rb:62]: nice, later on we could make this a flash for an error

File: app/controllers/merchants/items_controller.rb [hunk 2]
```diff
@@ -4,9 +4,58 @@ def index
...
   end
 
+  def new
+    @merchant = User.find(current_user.id)
+    @item = @merchant.items.new
+  end
+
+  def create
+    if verify_price(params) == false
+      flash[:notice] = "The price for that item cannot be negative."
```
Comment (WHomer) [inline app/controllers/merchants/items_controller.rb:14]: nevermind, i see it here.

**Review by WHomer — APPROVED** [review]
nice.

**Review by WHomer — APPROVED** [review]
looks good.
